src/test/regress/expected/with_modifying.out

```diff
@@ -193,9 +193,8 @@ WITH cte AS (
 	SELECT user_id, value_2 FROM users_table WHERE value_2 IN (1, 2)
 )
 INSERT INTO modify_table (SELECT (SELECT value_2 FROM cte GROUP BY value_2));
-WARNING:  more than one row returned by a subquery used as an expression
+ERROR:  more than one row returned by a subquery used as an expression
```

Comment:
Hmm, when you open the PR against master, do you think would the error message change back? Similar for the below test.

Anyway, I'll need to handle bunch of those things on #2762